src/test/ui/async-keyword/2015-edition-warning.rs

```diff
@@ -0,0 +1,13 @@
+#![allow(non_camel_case_types)]
+#![deny(keyword_idents)]
+
+mod outer_mod {
```

Comment:
It'd be good to make this a "rustfix" test, just to be sure.  Just add `// run-rustfix` and then re-run with `--bless`. This will make a `.fixed` file to sure the effect of the suggestions.

Reply:
Done.